go-selinux/selinux_linux.go

```diff
@@ -548,37 +548,35 @@ func rangeStrToMLSRange(rangeStr string) (*mlsRange, error) {
 
 // bitsetToStr takes a category bitset and returns it in the
 // canonical selinux syntax
-func bitsetToStr(c *bitset.BitSet) string {
+func bitsetToStr(c *big.Int) string {
 	var str string
-	i, e := c.NextSet(0)
-	len := 0
-	for e {
-		if len == 0 {
+
+	i := int(c.TrailingZeroBits())
+	length := 0
+	for i < c.BitLen() {
```

Comment:
nit: this could be a `for init; condition; increment` loop

Reply:
done